flight/Modules/Attitude/naze32/attitude.c

```diff
@@ -0,0 +1,730 @@
+/**
```

Comment:
Yes.  This is very similar to the CC3D attitude code.  It's unfortunate the amount of duplication between the CC3D code, this code, and the main attitude code.  They're different because CC3D and this code talk directly to the sensors and there's some differences.  As part of target cleanup I'm going to work in this area (hopefully getting some additional functionality for CC3D in the process).